curriculum/challenges/english/25-front-end-development/workshop-nutritional-label/615f7de4487b64919bb4aa5e.md

```diff
@@ -56,15 +56,15 @@ const zeroPercentSpan = spans.filter(span => span?.innerHTML?.match(/^[\s\n]*0%[
 assert(zeroPercentSpan.length === 1);
 ```
 
-Your `0%` `span` should have the `class` attribute set to `bold`.
+The `span` element that contains the text `0%` should have its `class` attribute set to `bold`.
 
 ```js
 const spans = [...document.querySelector('.daily-value.small-text')?.lastElementChild?.querySelectorAll('span')];
 const zeroPercentSpan = spans.filter(span => span?.innerHTML?.match(/^[\s\n]*0%[\s\n]*$/));
 assert(zeroPercentSpan[0]?.classList?.contains('bold'));
 ```
 
-Your `Cholesterol` `span` and your `0mg` text should be wrapped in a `span`.
+The text `Cholesterol` should be in a `span` with the class `bold`, and both `Cholesterol` and `0mg` should be wrapped together in another `span` element.
```

Comment:
"The text `Cholesterol` should be in a `span` with the class `bold`" is already part of an other hint, we should not repeat it here


```suggestion
The `span` element that contains the text `Cholesterol` and the text `0mg` should be wrapped together in another `span` element.
```